loopout.go

```diff
@@ -306,6 +306,15 @@ func (s *loopOutSwap) sendUpdate(ctx context.Context) error {
 
 	info.HtlcAddressP2WSH = s.htlc.Address
 
+	// In order to avoid potentially dangerous ownership sharing
+	// we copy the outgoing channel set.
+	if s.OutgoingChanSet != nil {
+		outgoingChanSet := make(loopdb.ChannelSet, len(s.OutgoingChanSet))
+		copy(outgoingChanSet[:], s.OutgoingChanSet[:])
+
+		info.OutgoingChanSet = outgoingChanSet
+	}
+
```

Comment:
This could also be done like this:
```go
if s.LastHop != nil {
    lastHop, err := route.NewVertexFromBytes(s.LastHop[:])
    if err != nil {
	    return err
    }
    
    info.LastHop = &lastHop
}
```
any preferences?

Reply:
same logic above. We only do this to copy values, not check their validity.